file_paths.md

```diff
@@ -163,39 +163,107 @@ and `pxp-agent`, all running as `LocalSystem` by default. It will not
 create a `puppet` user or group.
 
     Path                                      Setting                        3.x
-    C:\ProgramData                                                           n/a
+    C:\ProgramData\PuppetLabs *                                              n/a
 
     C:\ProgramData\PuppetLabs\client-tools    # default client tool settings
         puppet-access.conf *
         puppet-orchestrator.conf *
 
-
     C:\ProgramData\PuppetLabs\code *          # :codedir                     contents moved from C:\ProgramData\PuppetLabs\puppet\etc (:confdir)
         environments *                        # :environmentpath
           production *
             environment.conf *
+            hieradata *
             manifests *
             modules *
-        hiera.yaml *                          # :hiera_config
-        hieradata *                           # n/a
-        modules *                             # user modulepath
+        hiera.yaml                            # :hiera_config
+        modules                               # user modulepath
+
+    C:\ProgramData\PuppetLabs\mcollective *
+        client.cfg *
+        facts.yaml *
+        server.cfg *
 
-    C:\ProgramData\PuppetLabs\facter *                                       same
+    C:\ProgramData\PuppetLabs\puppet          # :confdir
+        auth.conf *                           # :rest_authconfig
+        autosign.conf                         # :autosign
+        binder_config.yaml                    # :binder_config
+        csr_attributes.yaml                   # :csr_attributes
+        custom_trusted_oid_mapping.yaml       # :trusted_oid_mapping_file
+        device.conf                           # :deviceconfig
+        fileserver.conf                       # :fileserverconfig
+        puppet.conf *                         # :config
+        routes.yaml                           # :route_file
+        ssl                                   # :ssldir
+
+    C:\ProgramData\PuppetLabs\pxp-agent
+        modules *                             # stores configuration files for pxp-agent modules
+            pxp-module-puppet.conf *          # configuration file of the pxp module puppet (optionally, to override puppet.bat location)
+        pxp-agent.conf                        # pxp-agent configuration file
+
+    C:\Program Files\Puppet Labs\bin *
```

Comment:
For strange reasons, only administrators running elevated can create symlinks on windows, so that could be problematic in non-admin installs. Fortunately we don't need to worry about 2003 (as that doesn't support symlinks).